plugins/core/src/main/kotlin/de/fayard/refreshVersions/core/internal/DependencyGroup.kt

```diff
@@ -27,6 +27,7 @@ open class DependencyGroup(
         val ALL_RULES: List<ArtifactVersionKeyRule>
             get() = ALL.mapNotNull { it.rule }
 
+        var disableBomCheck: Boolean = false
```

Comment:
Shouldn't this property be `internal`, or `@InternalRefreshVersionsApi`?

Reply:
done, ,thanks